extension/android/benchmark/app/src/main/java/org/pytorch/minibench/BenchmarkActivity.java

```diff
@@ -46,18 +46,21 @@ protected void onCreate(Bundle savedInstanceState) {
       stats.latency.add(forwardMs);
     }
 
-    // TODO (huydhn): Remove txt files here once the JSON format is ready
```

Comment:
I should probably log the time for module.loadMethod() before first forward() 😢

Reply:
I see, let me try to copy it from llama and add one here too